### ISSUE TITLE
aria-role rule: only lowercase values should be allowed.

### ISSUE THREAD

**Issue by jessebeach** [evcohen/eslint-plugin-jsx-a11y#472]
Currently, the following "pass", but should be considered invalid.

```
role="Button"
role="Presentation"
```
